main.py

```diff
@@ -72,6 +77,26 @@ def resend_stored_flows(self, event):
             self.resent_flows.add(dpid)
             log.info(f'Flows resent to Switch {dpid}')
 
+    def is_ignored(self, flow):
+        """Verify if the flow is in the exception range.
+
+        Check by `cookie` range and `table_id` range.
+        """
+        # Check by cookie
+        if len(self.cookie_exception_range) == 2:
+            begin_cookie = self.cookie_exception_range[0]
+            end_cookie = self.cookie_exception_range[1]
+            if flow.cookie >= begin_cookie and flow.cookie <= end_cookie:
+                return True
+
+        # Check by `table_id`
+        if len(self.table_id_exception_range) == 2:
+            begin_tab_id = self.table_id_exception_range[0]
+            end_tab_id = self.table_id_exception_range[1]
+            if flow.table_id >= begin_tab_id and flow.table_id <= end_tab_id:
+                return True
```

Comment:
Suggestion: new function to check exception ranges.

Reply:
The exceptions in 'settings.py' could be a list that contains a tuple (to represent a range o values) or a single integer value. This would allow more flexibility in the exceptions, i.e. allow setting more than one range of exceptions and single values.

Reply:
At commit 1477b0e, a function to do this was added.